### PR TITLE
Adding user management page. Builds on top of code from PR #248

### LINKED ISSUE #251: Adding a user management page

### PR DESCRIPTION
**Pull request by haroonmoh**
This PR fixes #251

### CODE REVIEW

File: control-plane/src/button-clicks.js [hunk 1]
```diff
@@ -74,58 +74,6 @@ $('#updated').on('change', function () {
     }
 })
 
-$('#filter-jobs-button').on("click", function () {
```
Comment (taoliaoleo) [inline control-plane/src/button-clicks.js:77]: Are we moving this block of code for implementing new feature or for cleaning up? If for the later, I'd recommend doing it in a separate issue/pr.

If there's too much work to separate, we can keep it in this pr this time. But we should clearly describe which parts are for cleaning up and which for new feature.

**Review by taoliaoleo — COMMENTED** [review]
Did you get a chance to look for replacing the images with links?

**Comment by hostirosti** [comment]
Did you try to use `git rebase` instead of `git merge`? You might have to create a patch file against main to have a clean commit from all your changes.

**Comment by haroonmoh** [comment]
> 

I have rebased this branch

**Pull request merged**